pages/common/openttd.md

```diff
@@ -0,0 +1,28 @@
+# openttd
+
+> Open source clone of the Microprose game "Transport Tycoon Deluxe".
+> More information: <https://www.openttd.org>.
+
+- Load new/save game at start:
+
+`openttd -g {{path/to/file}}`
+
+- Start with the specified window resolution:
+
+`openttd -r {{1920x1200)}}`
```

Comment:
```suggestion
`openttd -r {{1920x1080)}}`
```

Reply:
It the closing parenthesis after `1080` a typo?

Reply:
Is the parenthesis after `1080` a typo?

Reply:
Seems so indeed ;]

Reply:
Good catch! Updated.